docs/advanced-configuration.md

```diff
@@ -191,6 +191,75 @@ for the Fargate distribution has two primary differences between regular `eks` t
     node label. The Collector's ClusterRole for `eks/fargate` will allow the `patch` verb on `nodes` resources for the default API groups to allow the cluster
     receiver's init container to add this node label for designated self monitoring.
 
+## Control Plane metrics
+
+By setting `autodetect.controlPlane=true` the helm chart will set up the otel-collector agent to collect metrics from
+the control plane.
+
+To collect control plane metrics, the helm chart has the otel-collector agent on each node use the
+[receiver creator](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md)
+to instantiate control plane receivers at runtime. The receiver creator has a set of discovery rules to know
+which control plane receivers to create. The default discovery rules can vary depending on the Kubernetes distribution
+and version. If your control plane is using non-standard specs, then you can provide a custom configuration so the
+otel-collector agent can still successfully connect.
```

Comment:
Could be helpful to say "described below" with a link to the header.

Reply:
Done